### ISSUE TITLE
Mark the posts that you've added

### ISSUE THREAD

**Issue by uesteibar** [retro-tool/retro-tool#51]
It would be useful to somehow mark the posts that you yourself have added. Of course only you would be able to see what you have added, for others this would be anonymous.

We already identify your posts on the first phase so this shouldn't be a big deal, but I'd like to see what you @retro-tool/all think :)

**Comment by atrzaska** [comment]
I'm ok with having them marked but I'd preffer something subtle.

**Comment by rudeayelo** [comment]
Agree with Andrzej, I'd totally do it, but subtly

**Comment by uesteibar** [comment]
@iamsilesc any design ideas?

**Comment by marciobarrios** [comment]
Agree with Rude and Andrzej.

**Comment by iamsilesc** [comment]
By posts, you mean topics on each column? If so, I think that we can come up with an easy design. I also see it as a benefit in the phase of going on each topic, during the retro. This could avoid the typical question: _Who wrote this?_  (Sorry, I'm not sure if I'm on the same page as you).

**Comment by uesteibar** [comment]
> By posts, you mean topics on each column?

yes, I mean a retro item :)

---

> This could avoid the typical question: Who wrote this?

I am proposing that only _you_ can see which ones were posted by you, otherwise we'd lose the anonymity

**Comment by iamsilesc** [comment]
> I am proposing that only _you_ can see which ones were posted by you, otherwise we'd lose the anonymity

Yes, I agree. What about during the retro? Here I see it different, because you want to know who wrote X/Y, no?  Just saying because I saw it in many tools already, that you can see the owner of the topics. Maybe it's something we need to research a bit.

Let's start with the _only you can see._

**Comment by uesteibar** [comment]
I would prefer not to let others see who has posted what, I personally would prefer to keep it anonymous. If a team wants to make it non-anonymous, they can a) add their name in the post (I know teams that do this) or b) say it out loud when reviewing the topics.

It would also mean we'd have to add some kind of login/identification of users, and one of the main features of the app is that we don't do that. Maybe we could allow people to just enter a "name"?


anyways, I'm just thinking aloud here, I'd start with _only you can see_ yes 👍

**Comment by marciobarrios** [comment]
@iamsilesc have in mind that our tool is completely anonymous and doesn't require any login, so we can only identify your own items.

I completely agree with @uesteibar on this, we should keep being anonymous and avoid any login.
